assets/src/shared/lib/utils.ts

```diff
@@ -4,3 +4,25 @@ import { twMerge } from 'tailwind-merge';
 export function cn(...inputs: ClassValue[]) {
   return twMerge(clsx(inputs));
 }
+
+const default_name_len =
```

Comment:
this is utils file why constant in this place?
see `shared/const` folder for example. And use UPPER_SNAKE_CASE for constants, it is a code style in the project

Reply:
and why why why and what does this function do?

Reply:
hm, fix 4c2e38365cf2e53bde7a6f8638430da6a10ab178

I guess I'll just put the constant inside the function, it shouldn't be reused

Reply:
> and why why why and what does this function do?

trim too long rows